tests/testthat/test-api.R

```diff
@@ -0,0 +1,58 @@
+test_that("create_bugzilla_key works", {
+    vcr::use_cassette("create_bugzilla_key", {
+        cbk <- create_bugzilla_key(missing_host())
+        url <- "https://bugs.r-project.org/bugzilla/userprefs.cgi?tab=apikey"
+    })
+    expect_equal(paste0(missing_host(), "userprefs.cgi?tab=apikey"), url)
+    cli::test_that_cli("cli_alert_warning works", {
```

Comment:
Update all the tests replacing the test_that call with the `test_that_cli` whenever you want to capture the output of cli.

Reply:
Sure! I'll do that.